persistent/test/Database/Persist/QuasiSpec.hs

```diff
@@ -135,6 +135,15 @@ spec = describe "Quasi" $ do
                         ]
                     )
 
+        it "handles numbers" $
+            parseLine "  one (Finite 1)" `shouldBe`
+                Just
+                    ( Line 2
+                        [ Token "one"
+                        , Token "Finite 1"
+                        ]
+                    )
```

Comment:
This was an initial test I did as an exploratory thing to verify the parser was even parsing numbers at all to begin with (which it was), this test could potentially be removed now. Though it may also be worthwhile leaving in.

Reply:
Yeah! It's good to have around as a regression. Thanks for the test!